bridge/client/sdp.js

```diff
@@ -379,6 +387,11 @@ if (typeof(SDP) == "undefined")
         var sdpText = fillTemplate(templates.sdp, sdpObj);
         sdpText = fillTemplate(sdpText, sdpObj.originator);
 
+        if (!sdpObj.noTrickle)
+            var iceTrickleLine = "a=ice-options:trickle\r\n";
+
+        sdpText = fillTemplate(sdpText, { "trickleLine": iceTrickleLine });
```

Comment:
Sure there is a way to make it clearer and more explicit, in fact this was not how I did it first. But then I looked at how the msidSemanticLine is handled, and modeled this one after that to be consistent.